sda-auth/keys/c4gh_key.pub.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CRYPT4GH PUBLIC KEY-----
+J75CRF/Z45yb455rNqVeYeOAH8hF9jeBqBPT/pl34Xo=
+-----END CRYPT4GH PUBLIC KEY-----
```

Comment:
```suggestion
```
Don't commit public/private keys to the repo.
If they are needed for testing have a script or container generate them dynamically.